cloudcompose/cluster/cloudinit.py

```diff
@@ -10,11 +10,10 @@ def __init__(self, cloud_config, base_dir='.'):
         self.template_file = 'cluster.sh'
 
     def build(self, **kwargs):
-        config_data = self.cloud_config.config_data('cluster')
+        config_data = self.cloud_config.config_data('cluster')[0]
```

Comment:
Okay the index change makes sense. Can you make the subnets a YAML list instead of comma separated string?